courses/free-courses-en.md

```diff
@@ -1365,6 +1365,7 @@
 
 ### Scala
 
+* [Effective Programming in Scala]([https://www.coursera.org/course/reactive](https://www.classcentral.com/course/effective-scala-39307))
```

Comment:
one link only please.
when you've made changes, please add `(changes made)` to the title of your PR